### PR TITLE
Open OpenFlow port when SDN controller is set/close it when SDN controller unset

### PR DESCRIPTION
**Pull request by benjamreis**

### CODE REVIEW

**Comment by benjamreis** [comment]
build & xapi tests ok

File: SPECS/xapi.spec [hunk 1]
```diff
@@ -424,6 +425,9 @@ Coverage files from unit tests
 %endif
 
 %changelog
+* Fri Apr 24 2020 Benjamin Reis <benjamin.reis@vates.fr> - 1.214.1-2.7
+- Patch xapi-1.214.1-open-openflow-port.XCP-ng.patch modified
+
```
Comment (stormi) [inline SPECS/xapi.spec:430]: Add a line telling what the change brings

File: SOURCES/xapi-1.214.1-open-openflow-port.XCP-ng.patch [hunk 2]
```diff
@@ -0,0 +1,24 @@
+Open OpenFlow port (TCP 6653) when a SDN controller is set and close ot when SDN controller is unset.
```
Comment (stormi) [inline SOURCES/xapi-1.214.1-open-openflow-port.XCP-ng.patch:1]: s/close ot/close it/

**Comment by stormi** [comment]
We don't use the `feat(scripts/plugins/openvswitch-config-update)` convention in the RPMs repositories. Can you also limit the first line of the comment to 72 characters, and add details if needed under it?

**Comment by benjamreis** [comment]
I'll rebase and change the commit name once the review is accepted. :)

**Comment by stormi** [comment]
Did you push the new commit?

**Comment by benjamreis** [comment]
Yes... On the wrong PR... for 8.0...

**Comment by stormi** [comment]
The other one I'll review only when we've pushed that one :)

**Comment by benjamreis** [comment]
Pushed :)

**Comment by stormi** [comment]
Looks good

**Pull request merged**